alembic/versions/ae776dc8160b_add_koji_task_analysis_table.py

```diff
@@ -0,0 +1,121 @@
+"""Add koji_task_analysis table
+
+Revision ID: ae776dc8160b
+Revises: 2f60bb92411f
+Create Date: 2025-01-13 12:00:00.000000
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision: str = "ae776dc8160b"
+down_revision: Union[str, None] = "2f60bb92411f"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # Create the KojihubInstance enum
+    kojihub_instance_enum = sa.Enum(
+        "https://brewhub.engineering.redhat.com/brewhub",
+        "https://cbs.centos.org/kojihub/",
+        "https://koji.fedoraproject.org/kojihub",
+        "https://s390.koji.fedoraproject.org/kojihub",
+        "https://koji.stg.fedoraproject.org/kojihub",
+        "https://kojihub.stream.rdu2.redhat.com/kojihub",
+        name="kojihubinstance",
+    )
+    kojihub_instance_enum.create(op.get_bind())
+
+    # Create the koji_task_analysis table
+    op.create_table(
+        "koji_task_analysis",
+        sa.Column("id", sa.Integer(), nullable=False),
+        sa.Column(
+            "koji_instance",
+            sa.Enum(
+                "https://brewhub.engineering.redhat.com/brewhub",
+                "https://cbs.centos.org/kojihub/",
+                "https://koji.fedoraproject.org/kojihub",
+                "https://s390.koji.fedoraproject.org/kojihub",
+                "https://koji.stg.fedoraproject.org/kojihub",
+                "https://kojihub.stream.rdu2.redhat.com/kojihub",
```

Comment:
Would it make sense to set these values in a separate variable and use it both cases?

Reply:
Yeah, we can simplify this. I had this code generated by alembic and didn't realize it duplicated that.

Reply:
This actually got completely replaced in the latest changes, as I realized that we need the set of Koji instances to be variable, since there are some out there -- like Datto -- whose [internal] URLs I don't know and therefore can't hard-code.

In the latest iteration, this is set in the config.yml and saved to the Database.